main.js

```diff
@@ -1,26 +1,32 @@
 var phrase = 'grumpy wizards make toxic brew for the evil queen'
 var array = []
 var $phraseContainer = document.getElementsByTagName('div')[1]
+
 for(var c = 0; c < phrase.length; c++) {
   var obj = {
     letter: ''
   }
   obj.letter = phrase[c]
   array.push(obj)
 }
+array.push({
```

Comment:
Instead of mixing your characters and the current index, use an object to hold your app state.

```js
var appState = {
  characters: [], // <-- the objects go here
  currentCharacterIndex: 0 // <-- this is which index we're on
}
```

Also note that there are no names like `obj` or `array`. Everything's name should describe its purpose as much as possible.